Correcao Após Duvidas Respondidas com Suporte Itau

joaojesusvictor opened the pull request.

joaojesusvictor committed:
Correcao Após Duvidas Respondidas com Suporte Itau

B010 commented on src/CnabContasPagar/CnabContasPagar/Bancos/BancoItau240.cs:
```diff
@@ -188,47 +188,51 @@ public void DetalheA(StringBuilder b, Liquidacao liquidacao)
             {
                 b.AppendTexto(5, "00005"); // Finalidade da TED (Pgto Fornecedores)
             }
+            else
```
só coloca um comentário no `else `falando quando normalmente cai nele

joaojesusvictor replied:
Feito

B010 commented on src/CnabContasPagar/CnabContasPagar/Bancos/BancoItau240.cs:
```diff
@@ -188,47 +188,51 @@ public void DetalheA(StringBuilder b, Liquidacao liquidacao)
-            b.Append(new string(' ', 10));
-            b.Append(Environment.NewLine);
-        }
+        //public void DetalheANF(StringBuilder b, Liquidacao liquidacao)   // ESSE BLOCO NÃO TINHA                  NÃO APAGAR ESSE BLOCO
+        //{
+        //    b.Append("341"); //01-03
+        //    b.AppendNumero(4, codigoLote); //04-07
+        //    b.Append('3');
+        //    b.AppendNumero(5, codigoDetalhe);
+        //    b.Append('A');
+        //    b.Append("000"); //TIPO DE MOVIMENTO (000 = Inclusão de pagamento)
+        //    b.AppendNumero(3, 0);
+        //    b.AppendNumero(3, liquidacao.BancoFavorecido);
+        //    b.AppendTexto(20, FazerAgenciaContaFavorecido(liquidacao));
+        //    b.AppendTexto(30, liquidacao.NomeFavorecido);
+        //    b.AppendTexto(20, liquidacao.Documento); // Seu Numero
+        //    b.AppendData(liquidacao.DataPagamento);
+        //    b.Append("REA");
+        //    b.AppendNumero(15, 0);
+        //    b.AppendDinheiro(15, liquidacao.ValorPagamento);
+        //    b.Append(new string(' ', 20)); // Nosso Numero
+        //    b.AppendNumero(8, 0);
+        //    b.AppendNumero(15, 0);
+        //    b.AppendNumero(14, 0);
+        //    b.Append(new string(' ', 6));
+        //    b.AppendNumero(6, 0);
+        //    b.AppendNumero(14, liquidacao.CpfCnpjFavorecido);
+        //    b.AppendNumero(1, 0); // Tipo de Identificação
+        //    b.Append(new string(' ', 11));
+        //    b.Append('0');
+        //    b.Append(new string(' ', 10));
+        //    b.Append(Environment.NewLine);
+        //}
```
Deixa o método lá,  só coloca um comentário acima dizendo que ainda não foi implementado o pagamento por nota fiscal.

joaojesusvictor replied:
Feito

joaojesusvictor committed:
Correcao Validacao DV CodBarras

joaojesusvictor committed:
Add coments

joaojesusvictor committed:
Correcao na distribuicao do Arquivo

Falei com o Rodrigo e ele disse que o Segmento J-52-PIX é usado somente quando a forma de pagto do Boleto for por PIX

joaojesusvictor committed:
Correcao contagem de linhas p Boleto

joaojesusvictor committed:
Correcao Contagem Linhas do Lote do Arquivo Remessa

joaojesusvictor committed:
Merge remote-tracking branch 'upstream/main' into ContasPagar

joaojesusvictor committed:
Inicio Desenvolvimento para Bradesco

joaojesusvictor committed:
Evolucao Bradesco 500

joaojesusvictor committed:
Duvidas Solucionadas

joaojesusvictor committed:
Finalizacao Cnab Bradesco

Desenvolvimento arquivo cnab bradesco

joaojesusvictor committed:
Ajuste Layout Bradesco

- Inclusao do Numero Convenio

joaojesusvictor committed:
Ajustes Cnab Bradesco

- Ajustes na formatação do DigitoCC e da Data do Desconto na construção do arquivo Cnab do Bradesco

joaojesusvictor committed:
Inclusao Real Time

- Inclusao da Forma de Pagto Real Time para Bradesco

joaojesusvictor committed:
Melhorias Geracao Cnab Pagar

- Mudanças feitas para melhorias na geração do Arquivo Cnab Pagar

B010 committed:
Ajustes

joaojesusvictor committed:
Incluindo Msg Erro

- Msg erro no relatorio de retorno

B010 committed:
rtydmnxhfngxhfbng

B010 committed:
Atualizar
